waterlog/src/ConfigRedux/enhancers/monitorReducer.js

```diff
@@ -19,4 +19,4 @@ const monitorReducerEnhancer = createStore => (
     return createStore(monitoredReducer, initialState, enhancer)
 }
 
-export default monitorReducerEnhancer
+export default monitorReducerEnhancer
```

Comment:
empty line EOF and in `waterlog/src/ConfigRedux/middleware/logger.js` as well

Reply:
left empty line